recipes/qimage2ndarray/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: qimage2ndarray
+  version: "1.5.1"
+
+source:
+  fn: qimage2ndarray-1.5.1.tar.gz
+  url: https://pypi.python.org/packages/source/q/qimage2ndarray/qimage2ndarray-1.5.1.tar.gz
+  md5: 73743819ccf6043f237c6b7f63732674
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
If we use setuptools, it is rare to not need to do ```python setup.py install --single-version-externally-managed --record=record.txt```. Has that been intentionally omitted, or are you happy to adopt that here?